docs/Validation.md

```diff
@@ -420,6 +420,36 @@ Demonstration of the different behaviour:
 
 ## xml.validation.schema.enabled
```

Comment:
Title should read "xml.validation.namespaces.enabled"

Reply:
good catch, fixed.